src/components/AccountContainer.js

```diff
@@ -6,59 +6,51 @@ class AccountContainer extends Component {
 
   constructor() {
     super()
-
-    // we have provided this default state for you,
-    // use this to get the functionality working
-    // and then replace the default transactions with a call to the API
-
     this.state = {
       searchTerm: '',
-      transactions: [
-        {
-          id: 1,
-          posted_at: "2017-02-28 11:00:00",
-          description: "Leather Pants, Gap co.",
-          category: "Fashion",
-          amount: -20000
-        },
-        {
-          id: 2,
-          posted_at: "2017-02-29 10:30:00",
-          description: "Paycheck from Bob's Burgers",
-          category: "Income",
-          amount: 100000
-        },
-        {
-          id: 3,
-          posted_at: "2017-05-24 10:53:00",
-          description: "'Pair Programming Illuminated' by Laurie Williams and Robert Kessler",
-          category: "Book",
-          amount: 1498
-        },
-        {
-          id: 4,
-          posted_at: "2017-05-24 08:52:00",
-          description: "Medium Iced Cold Brew, Gregory's Coffee",
-          category: "Coffee",
-          amount: 365
-        }
-      ]
+      transactions: []
     }
   }
+    componentDidMount () {
+      let parseJson = function (response) { return response.json();
+      };
+
+      fetch('https://boiling-brook-94902.herokuapp.com/transactions',{
+        method: 'GET',
+        headers: {'Accept': 'application/json'}})
+      .then(parseJson)
+      .then( (response)=> this.setState(
+      {
+        transactions: response
+      }
+    )
+  )
+}
 
-  handleChange(event) {
-    // your code here
-  }
+    handleChange(event) {
+      let searchTerm = event.target.value
+
+      this.setState({
+        searchTerm,
+        transactions: this.state.transactions.filter(
```

Comment:
We discussed this in lecture, but you're overriding all of your transactions here. You instead just want to do this calculation and pass that filtered list down into a new `TransactionList`